tests/php/sync/test_class.jetpack-sync-callables.php

```diff
@@ -1216,25 +1216,23 @@ public function test_sync_does_not_send_updates_if_array_order_is_only_change()
 	}
 
 	/**
-	 * Randomize the get_plugins array keys.
+	 * Reorder the get_plugins array keys.
 	 *
 	 * @param array $plugins array of plugins.
 	 *
 	 * @return array
 	 */
-	public function randomize_array_keys( $plugins ) {
+	public function reverse_array_keys( $plugins ) {
 		// First plugin in array.
 		$plugin_key = array_keys( $plugins )[0];
 
-		// sort the plugin's array entries.
-		$keys = array_keys( $plugins[ $plugin_key ] );
-		shuffle( $keys );
-		$random = array();
-		foreach ( $keys as $key ) {
-			$random[ $key ] = $plugins[ $plugin_key ][ $key ];
-		}
-		$plugins[ $plugin_key ] = $random;
-		return $plugins;
+		// reverse the plugin's array entries.
+		$keys                   = array_keys( $plugins[ $plugin_key ] );
+		$keys                   = array_reverse( $keys );
+		$plugins[ $plugin_key ] = $keys;
```

Comment:
This seems to not be doing the right thing. Just
```suggestion
		$plugins[ $plugin_key ] = array_reverse( $plugins[ $plugin_key ] );
```
should be sufficient.

Reply:
🎩 I know better than to quickly make changes at random hours. Thanks for the thorough look over.